src/npc.cpp

```diff
@@ -739,7 +740,7 @@ int NpcScriptInterface::luaGetNpcCid(lua_State* L)
 int NpcScriptInterface::luaGetNpcParameter(lua_State* L)
 {
 	// getNpcParameter(paramKey)
-	Npc* npc = tfs::lua::getScriptEnv()->getNpc();
+	auto npc = tfs::lua::getScriptEnv()->getNpc();
```

Comment:
Could this be `const auto`
```suggestion
	const auto npc = tfs::lua::getScriptEnv()->getNpc();
```